From 474c436aebe260b0e6e7042ce5291137b7c87b57 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Sat, 22 Feb 2014 17:22:53 +0100
Subject: [PATCH 119/182] ahci_platform: Drop support for ahci-strict platform
 device type

I've done a grep over the entire kernel tree and nothing is using this
(anymore?).

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Tejun Heo <tj@kernel.org>
---
 drivers/ata/ahci_platform.c |   11 -----------
 1 file changed, 11 deletions(-)

--- a/drivers/ata/ahci_platform.c
+++ b/drivers/ata/ahci_platform.c
@@ -31,7 +31,6 @@ static void ahci_host_stop(struct ata_ho
 enum ahci_type {
 	AHCI,		/* standard platform ahci */
 	IMX53_AHCI,	/* ahci on i.mx53 */
-	STRICT_AHCI,	/* delayed DMA engine start */
 };
 
 static struct platform_device_id ahci_devtype[] = {
@@ -42,9 +41,6 @@ static struct platform_device_id ahci_de
 		.name = "imx53-ahci",
 		.driver_data = IMX53_AHCI,
 	}, {
-		.name = "strict-ahci",
-		.driver_data = STRICT_AHCI,
-	}, {
 		/* sentinel */
 	}
 };
@@ -75,13 +71,6 @@ static const struct ata_port_info ahci_p
 		.udma_mask	= ATA_UDMA6,
 		.port_ops	= &ahci_platform_retry_srst_ops,
 	},
-	[STRICT_AHCI] = {
-		AHCI_HFLAGS	(AHCI_HFLAG_DELAY_ENGINE),
-		.flags		= AHCI_FLAG_COMMON,
-		.pio_mask	= ATA_PIO4,
-		.udma_mask	= ATA_UDMA6,
-		.port_ops	= &ahci_platform_ops,
-	},
 };
 
 static struct scsi_host_template ahci_platform_sht = {
